### DISCUSSION TITLE
FrozenError: can't modify frozen String

### DISCUSSION THREAD

**Discussion by Itseric100** [postalserver/postal#2115] [Help with using Postal]
Does anyone know to to go about fixing this problem i am facing please, i have tried every possible knowledge known and applicable to it
![Screenshot 2022-08-15 220207](https://user-images.githubusercontent.com/96216385/184717847-83d32e73-a9b2-4eb8-b099-213b4a30fc53.png)
.

**Comment by Itseric100** [comment]
when i checked the log, this is what i got. And this is using the postal testing part to do this, so i didnt use any third party softw
![Screenshot 2022-08-15 223741](https://user-images.githubusercontent.com/96216385/184723377-8beca9cc-0bde-416e-aaf3-fe6d597e6682.png)
are for the testing.

**Comment by willpower232** [comment]
Are you running the latest Postal via docker? I don't think we've seen this error properly before.

**Reply by huynhphuong10284** [reply]
I also use docker with latest Postal, send out to other domains successfully. But sending internal domain has the same problem (ex: abx@domain.com to hehe@domain.com)
